packages/web-components/src/mixins/uswds-error-border.scss

```diff
@@ -3,7 +3,7 @@
 
 :host([error][uswds]:not([error=''])) {
     border-left: 0.4rem solid $v3-color-error-dark; /* USWDS red-60v, $theme-color-error-dark */
```

Comment:
Is it supposed to be all `rem`? If so, this was missed: `border-left: 0.4rem`

Reply:
That rem is a value which should be addressed with a formation_override entry, as it's specific to the v3 version.

Reply:
This is a style override of USWDS so that we can display our own error style. So what is `0.4rem` being based off of? The Formation base font or the USWDS base font? As it is now, it's not being addressed by either a formation override or a v3 override. 

If we're not sure which conversion function to pass it into (v1 or v3), then why don't we just directly change it to whatever pixel value our [Sketch file](https://www.sketch.com/s/610156b6-f281-4497-81f3-64454fc72156/p/447FB6DA-215E-4F15-A71B-94B7686DA614/canvas) says the error border width should be? (I'm seeing it as 4px):

![Screenshot 2023-04-28 at 11 50 57 AM](https://user-images.githubusercontent.com/872479/235207111-45e4a3d6-3a93-4329-9c92-e5aadd55a32c.png)

Reply:
With that said, I think we need to also change `padding-left: v3_rem_to_px(1rem);` to just `16px` (like the Sketch file also says).

Reply:
I'm good with updating to what the sketch file says.